pulp_2_tests/constants.py

```diff
@@ -520,6 +520,17 @@
 RPM_INVALID_URL = urljoin(RPM_INVALID_FEED_URL, 'invalid.rpm')
 """The URL to an invalid RPM package."""
 
+RPM_KICKSTART_FEED_URL = urljoin(PULP_FIXTURES_BASE_URL, 'rpm-kickstart/')
+"""The URL to a KICKSTART repository.
+
+NOTE:: This repository is not generated by `pulp-fixtures`_. Please see
```

Comment:
```suggestion
..NOTE:: This repository is not generated by `pulp-fixtures`_. Please see
```

See: https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#paragraph-level-markup